crux-core/src/crux/codec.clj

```diff
@@ -99,6 +100,9 @@
    (fn []
      (mem/->off-heap nil-id-bytes (mem/allocate-unpooled-buffer (count nil-id-bytes))))))
 
+(defn- sort-maps [o]
+  (->> o (walk/postwalk (fn [o] (cond->> o (map? o) (into (sorted-map)))))))
```

Comment:
This should probably be `instance? Map` as `map?` just checks for `IPeristentMap`? (Similar to how the change was done below for `id->buffer`.)